mobile/src/employee-details/employee-details-events-list.tsx

```diff
@@ -130,6 +130,9 @@ export class EmployeeDetailsEventsList extends Component<EmployeeDetailsEventsLi
                 const prefix = event.dates.endDate.isAfter(moment(), 'date') ? 'has coming ' : 'on ';
                 description = prefix + event.type.toLowerCase();
                 break;
+            case CalendarEventStatus.AccountingReady:
+                description = `has ${event.type.toLowerCase()} documents ready`;
```

Comment:
I'm afraid that it will overflow the text container. I can't check because I don't have configured development environment.

Reply:
Looks good on iPhone Xs and Meizu M5s